lib/kapnismology/result.rb

```diff
@@ -52,23 +52,29 @@ def initialize(passed, data, message)
       @passed = passed
       @data = data
       @message = message
-      @extra_messages = []
+      @debug_messages = []
     end
   end
 
-  # This class can be returned when a check do not want to assert if it passed or not.
-  # Instead it can return certain information about the check or the system.
-  class InfoResult < BaseResult
+  # Deprecated NullResult class provided for compatibility.
+  class NullResult < BaseResult
     def initialize(data, message = 'The result could not be determined')
       @passed = true
       @data = data
       @message = message
-      @extra_messages = []
+      @debug_messages = []
     end
 
-    # Redefining to have our own unique output
+    def to_s(name)
+      <<-eos
+#{Terminal.yellow('NullResult is deprecated.')}
```

Comment:
added this message to show when we run `rake smoke_test:all`

<img width="563" alt="screen shot 2016-07-29 at 1 14 09 pm" src="https://cloud.githubusercontent.com/assets/16286071/17237582/7c73bcbe-558e-11e6-96f5-2c92eca5ef94.png">